src/worker/worker.pb.gw.go

```diff
@@ -0,0 +1,342 @@
+// Code generated by protoc-gen-grpc-gateway. DO NOT EDIT.
```

Comment:
This API is for datum batching.  I don't think we serve this on the full `pachd`s.  If there's an easy way to avoid generating this, I would do that.

Reply:
@zmajeed 👆

Reply:
I think it's simplest to not exclude any protos from proto generators.  Today we are not calling any datum batching methods over plain HTTP, but someday that will be a feature request, and it will cost `figuring out how to not generate them today + figuring out how to not not generate them then + actually implement the plain-HTTP client`, whereas if we take no action today, then it costs `a few bytes in git + actually implementing the plain-HTTP client`.  For that reason, I say just generate everything for every proto.  If it's useful in the future, we win.  If it's not useful in the future, Github's cloud bill is slightly higher.  Meh!